packages/frontend/src/view/components/NetworkSelector.tsx

```diff
@@ -9,32 +20,103 @@ interface Props {
 }
 
 export function NetworkDropdownSelector(props: Props) {
-  const options = props.chainsToDisplay.map((chainToDisplay) => ({
-    label: ChainId.getName(chainToDisplay),
-    value: ChainId.getName(chainToDisplay),
-  }))
+  // We want almost instance user feedback, but api call
+  // and refresh itself will take a bit thus affecting ux
+  const [optimisticSelect, setOptimisticSelect] = useState<ChainId>(
+    props.chainId,
+  )
 
-  function onChange(option: DropdownOption) {
-    const chainId = ChainId.fromName(option.value)
+  function setChainId(chainId: ChainId) {
+    setOptimisticSelect(chainId)
     props.setChainId(chainId)
   }
 
-  const defaultValue = options.find(
-    (option) => option.value === ChainId.getName(props.chainId),
-  )
-
   if (!props.chainsToDisplay.includes(props.chainId)) {
-    props.setChainId(props.chainsToDisplay[0])
+    setChainId(props.chainsToDisplay[0])
   }
 
+  const pills = props.chainsToDisplay
+    .map((chainToDisplay, i) => ({
+      key: i,
+      icon: getIconForChain(chainToDisplay),
+      label: getPrettyChainName(chainToDisplay),
+      isActive: chainToDisplay === optimisticSelect,
+      onClick: () => setChainId(chainToDisplay),
+    }))
+    .sort((a, b) => a.label.localeCompare(b.label))
+    .map((props) => <PillSelector {...props} />)
+
   return (
-    <section className="bg-gray-900 px-6 pb-0 pt-6">
-      <label className="mb-4 text-xs text-gray-500">Select network</label>
-      <Dropdown
-        defaultValue={defaultValue}
-        onChange={onChange}
-        options={options}
-      />
+    <section className="bg-gray-500 px-6 py-8 md:justify-center">
```

Comment:
```suggestion
    <section className="bg-gray-500 p-6 md:justify-center">
```

Reply:
Let's also reduce the padding on mobile. WYT about making it sticky to the top (just like on L2BEAT product page)
<img width="388" alt="Screenshot 2023-11-21 at 11 42 41" src="https://github.com/l2beat/lz-monitoring/assets/38423054/b3f4d998-1fdc-414d-8ef6-dbf8bb929d5f">

Reply:
[feat: make network selector top-sticky](https://github.com/l2beat/lz-monitoring/pull/66/commits/61a5977e092da5b18b3c971dcca42136951a4daa)
[feat: refine selector paddings](https://github.com/l2beat/lz-monitoring/pull/66/commits/3dc4d905db10b1fc5de127cd6c95c32c30a4eb63)